client/service/service.go

```diff
@@ -107,6 +107,18 @@ func (s *Service) Create(spec CreateService) (created GetServiceSpec, err error)
 	if err != nil {
 		return
 	}
+	// The API will always clobber, which leads to odd behavior
+	// This aligns behavior with user expectations
+	// Don't clobber if the service name already exist
+	existing, err := s.GetByName(spec.Metadata.Name)
+	if err != nil {
+		return
+	}
+	if existing.ServiceID != "" {
+		err = fmt.Errorf("the service name %s is already exists", existing.ServiceName)
```

Comment:
Can we fix my typo here?
```suggestion
		err = fmt.Errorf("the service name %s already exists", existing.ServiceName)
```

Reply:
sure yeah i copied your message from role and policy.